### PR TITLE
[16.0][IMP] dms: Improve tests

### PR DESCRIPTION
**Pull request by victoralmau**
Changes done:
- Change `setup()` to `setUpClass()`
- Add BaseCommon
- Remove `multi_users` decorator (not used)
- Remove `_setup_test_data` and `_load` from common (not used)
- Change DirectoryMailTestCase (extends from StorageDatabaseBaseCase)
- Add `@mute_logger` to skip unnecessary unlink logs

@Tecnativa

### CODE REVIEW

File: dms/tests/test_benchmark.py [hunk 1]
```diff
@@ -2,25 +2,81 @@
...
-from .common import track_function
-
 _logger = logging.getLogger(__name__)
 
+# ----------------------------------------------------------
+# Decorators
+# ----------------------------------------------------------
+
+
+def track_function(
```
Comment (pedrobaeza) [inline dms/tests/test_benchmark.py]: Why moving this function?

**Review by pedrobaeza — COMMENTED** [review]
`multi_users` is not used anywhere?

File: dms/tests/test_benchmark.py (hunk 1 above)
Comment (victoralmau) [reply dms/tests/test_benchmark.py]: Because it is only used in this test.

**Comment by victoralmau** [comment]
> `multi_users`

No, `@users('user-1', 'user-2')` is used instead.

File: dms/tests/test_benchmark.py (hunk 1 above)
Comment (pedrobaeza) [reply dms/tests/test_benchmark.py]: Well, in this case, for reducing the diff, but also if in any moment it's needed by other test, let's keep it where it was. It's not an overhead having it there. It's loaded once the same being in one specific test or in the common one.

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
:+1:

**Pull request merged**